x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/index.tsx

```diff
@@ -178,6 +178,7 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
   thresholdFields,
   enableThresholdSuppression,
 }) => {
+  const { isSuppressionEnabled } = useAlertSuppression(ruleType);
```

Comment:
nit: can we keep name `isAlertSuppressionEnabled` instead of `isSuppressionEnabled`? all other suppression variables have alert prefix as the hook itself

Reply:
https://github.com/elastic/kibana/pull/175089/commits/dcca96391666c1d88c4825c295bf75bd1994c745